src/Microsoft.NetCore.Analyzers/UnitTests/Runtime/DoNotPassLiteralsAsLocalizedParametersTests.cs

```diff
@@ -1397,5 +1398,95 @@ private class DerivedClass : BaseClass
+        // No configuration - validate diagnostics in default configuration
+        [InlineData(@"")]
+        // Match by method name
+        [InlineData(@"dotnet_code_quality.excluded_symbol_names = M~M2")]
+        // Match by type name
+        [InlineData(@"dotnet_code_quality.excluded_symbol_names = C")]
+        // Match multiple methods by method documentation ID with "M:" prefix
+        [InlineData(@"dotnet_code_quality.excluded_symbol_names = M:C.M(string)~M:C.M2(C)~")]
+        // Match by type documentation ID with "T:" prefix
+        [InlineData(@"dotnet_code_quality.excluded_symbol_names = T:C")]
+        public void ShouldBeLocalized_MethodExcludedByConfiguration_NoDiagnostic(string editorConfigText)
+        {
+            var expected = Array.Empty<DiagnosticResult>();
+            if (string.IsNullOrEmpty(editorConfigText))
+            {
+                expected = new[]
+                {
+                    // Test0.cs(20,13): warning CA1303: Method 'void Test.M1(C c)' passes a literal string as parameter 'param' of a call to 'void C.M(string param)'. Retrieve the following string(s) from a resource table instead: "a".
+                    GetCSharpResultAt(20, 13, "void Test.M1(C c)", "param", "void C.M(string param)", "a"),
+                    // Test0.cs(21,14): warning CA1303: Method 'void Test.M1(C c)' passes a literal string as parameter 'param' of a call to 'void C.M2(string param)'. Retrieve the following string(s) from a resource table instead: "a".
+                    GetCSharpResultAt(21, 14, "void Test.M1(C c)", "param", "void C.M2(string param)", "a")
+                };
+            }
+
+            VerifyCSharp(@"
+using System.ComponentModel;
+
+public class C
+{
+    public void M([LocalizableAttribute(true)] string param)
+    {
+    }
+
+    public void M2([LocalizableAttribute(true)] string param)
+    {
+    }
+}
+
+public class Test
+{
+    public void M1(C c)
+    {
+        var str = ""a"";
+        c.M(str);
+        c.M2(str);
+    }
+}", GetEditorConfigAdditionalFile(editorConfigText), expected);
+        }
+
+        [Theory, WorkItem(2602, "https://github.com/dotnet/roslyn-analyzers/issues/2602")]
+        // No configuration - validate diagnostics in default configuration
+        [InlineData(@"")]
+        // Match by constructor name
+        [InlineData(@"dotnet_code_quality.excluded_symbol_names = .ctor")]
+        // Match by type name
+        [InlineData(@"dotnet_code_quality.excluded_symbol_names = Exception")]
+        // Match by namespace name
+        [InlineData(@"dotnet_code_quality.excluded_symbol_names = System")]
+        // Match by constructor documentation ID with "M:" prefix
+        [InlineData(@"dotnet_code_quality.excluded_symbol_names = M:System.Exception..ctor(System.String)")]
+        // Match by type documentation ID with "T:" prefix
+        [InlineData(@"dotnet_code_quality.excluded_symbol_names = T:System.Exception")]
```

Comment:
@x3ntrix @airbreather - This editorconfig setting should suffice your configuration scenario: `dotnet_code_quality.excluded_symbol_names = T:System.Exception`

Reply:
I don't see how. It looks like we need

    dotnet_code_quality.CA1303.excluded_symbol_names = T:System.Exception
    dotnet_code_quality.CA1303.excluded_symbol_names = T:System.ArgumentException
    dotnet_code_quality.CA1303.excluded_symbol_names = T:System.ArgumentNullException
    dotnet_code_quality.CA1303.excluded_symbol_names = T:System.InvalidOperationException
    ...

Reply:
Thanks, added a different option: https://github.com/dotnet/roslyn-analyzers/pull/2642/files#diff-0dc34fda02de9acf20e36dedb5fe1f26R1500

`dotnet_code_quality.excluded_type_names_with_derived_types = T:System.Exception`